### PR TITLE
WOE 10 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/johanns_stopgap.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Johann's Stopgap
+ManaCost:3 U
+Types:Sorcery
+K:Bargain
+S:Mode$ ReduceCost | ValidSpell$ Spell.Bargain | Type$ Spell | ValidCard$ Card.Self | Amount$ 2 | EffectZone$ All | Description$This spell costs {2} less to cast if it's bargained.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/johanns_stopgap.txt:5]: Space after "Description$"

File: forge-gui/res/cardsfolder/upcoming/johann_apprentice_sorcerer.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Johann, Apprentice Sorcerer
+ManaCost:2 U R
+Types:Legendary Creature Human Wizard
+PT:2/5
+S:Mode$ Continuous | Affected$ Card.TopLibrary+YouCtrl | AffectedZone$ Library | MayLookAt$ You | Description$ You may look at the top card of your library any time.
+S:Mode$ Continuous | MayPlay$ True | MayPlayLimit$ 1 | Affected$ Card.Instant+TopLibrary+YouCtrl,Card.Sorcery+TopLibrary+YouCtrl | AffectedZone$ Library | Description$ Once each turn, you may cast an instant or sorcery spell from the top of your library. (You still pay its costs. Timing rules still apply.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/johann_apprentice_sorcerer.txt:6]: nonLand

File: forge-gui/res/cardsfolder/upcoming/lich_knights_conquest.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Lich-Knights' Conquest
+ManaCost:4 B
+Types:Sorcery
+A:SP$ Sacrifice | Amount$ SacX | Optional$ True | SacValid$ Enchantment,Artifact,Card.token | RememberSacrificed$ True | SubAbility$ ReturnGrave | SpellDescription$ Sacrifice any number of artifacts, enchantments, and/or tokens. Return that many creature cards from your graveyard to the battlefield.
+SVar:ReturnGrave:DB$ ChangeZone | ChangeType$ Creature.YouOwn | Origin$ Graveyard | Destination$ Battlefield | ChangeNum$ ReturnX | Hidden$ True | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/lich_knights_conquest.txt:5]: Mandatory

File: forge-gui/res/cardsfolder/upcoming/kellans_lightblades.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Kellan's Lightblades
+ManaCost:1 W
+Types:Instant
+K:Bargain
+A:SP$ DealDamage | ValidTgts$ Creature.attacking,Creature.blocking | TgtPrompt$ Select target attacking or blocking creature | NumDmg$ 3 | SubAbility$ DBDestroy | ConditionCheckSVar$ Bargain | ConditionCompare$ EQ0 | SpellDescription$ CARDNAME deals 3 damage to target attacking or blocking creature. If this spell was bargained, destroy that creature instead.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/kellans_lightblades.txt:5]: ConditionCompare is the wrong param

**Review by tool4ever — APPROVED** [review]

**Pull request merged**